### DISCUSSION TITLE
How do I get an "empty" desktop?

### DISCUSSION THREAD

**Discussion by jd1500** [SpiralLinux/SpiralLinux-project#32] [Q&A] [answered]
Hello, I am a new SpiralLinux user (2 days).  Until recently I used Ubuntu Mate, which has a cool feature, and is the only thing I miss when I use SpiralLinux.  It's called "Mate Tweak" and allows all desktop icons to be hidden, so they can't be seen (see photo below), and makes for a very tidy desktop.

I like SpiralLinux (Mate) bettter than Ubu.Mate except for that one little  invisibility thing. 

Is there any way to make  Desktop Icons "invisible" like this;   I just click in the box and all icons are gone from Desktop:
![tweak](https://user-images.githubusercontent.com/108448715/176600344-fd52274c-25fd-41a3-82c9-b42e9b17a67d.png)

**Thank you in advance for your replies**

**Comment by geckolinux** [comment, marked as answer]
Hi there, thanks for your question.

The `mate-tweak` package can be installed via Synaptic if you want it. I've found that it never works well, which is why it's not included by default.

Another option would be to simply delete the only icon that SpiralLinux puts on the desktop after installation, which is the Language Support icon.

**Reply by jd1500** [reply]
Synaptic...Duh...I should have known that!   Lucky for me Tweak has always worked flawlessly on my installs.

Thank You!  That worked and is now installed on laptop...next task: Install SpiralLinux on Desktop. That solves the only issue I had with Spiral...You just gained a new user.

"Another option..."  Yes, I deleted that right away, but   I had a different issue...I keep my most used apps on the panel for easy access. Normally I just move the Icon to the panel and I'm done.  But unless I leave Thunderbird (email) on the desktop it won't stay locked to panel, but I still don't want to see it.  Tweak solves that problem.  

You will probably hear from me again, but for now, I am the proud owner of a shiny new SpiralLinux installation, and looking forward to the new adventure!

Thanks again!

**Reply by geckolinux** [reply]
Fantastic, glad it's working well for you!

**Reply by jd1500** [reply]
@geckolinux, I am wrritng this from my new  and improved Desktop using SpiralLinux!   Happy, Happy!

Just thought you might find this interesting or amusing...that problem I had with the "invisible Thunderbird" ?
Does not exist now, Tbird stays locked regardless if I run Tweak...Strange but awesome.

Thanks again for such a great OS!!
